Content.Client/Damage/Systems/StaminaSystem.cs

```diff
@@ -0,0 +1,7 @@
+﻿using Content.Shared.Damage.Systems;
+
+namespace Content.Client.Damage.Systems;
+
+public sealed class StaminaSystem : SharedStaminaSystem
```

Comment:
```suggestion
public sealed partial class StaminaSystem : SharedStaminaSystem
```
should be partial so that forks can easier extend it from another file
same for all the others